src/components/task/existing-card/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import propTypes from "prop-types";
+import dayjs from "dayjs";
+
+import Button from "../../base/button";
+import Image from "../../base/image";
+import Styles from "./index.module.scss";
+import TextArea from "../../base/text-area";
+
+function TaskCard({
+  title,
+  createdAt,
+  isCompleted,
+  completedAt,
+  isTaskOnEditMode,
+}) {
+  const [isTextAreaVisible, setIsTextAreaVisible] = useState(isTaskOnEditMode);
+
+  const calculateDateDifference = (startDate, endDate = dayjs()) =>
+    endDate.diff(startDate, "day");
```

Comment:
Is this functionality necessary for [ Step-2: Add functionality to Add tasks](https://trello.com/c/NnZeAh53/2-step-2-add-functionality-to-add-tasks)?

Our goal will be coding only the logical stuff that is required for the current step. So that we can have every step separated as well as the PR. 🚀